doc/src/site/sphinx/ml/sw_isolation_forest.rst

```diff
@@ -102,3 +102,97 @@ See also :ref:`parameters_H2OIsolationForest`.
         .. code:: python
 
             model.transform(testingDF).show(truncate = False)
+
+
+Train Isolation Forest with H2OGridSearch
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you're not sure about exact values for hyper-parameters of Isolation Forest, you can plug ``H2OIsolationForest`` to
+``H2OGridSearch`` and define a hyper-space to be walked through. Unlike other Sparkling Water algorithms used in
+``H2OGridSearch``, you must pass ``validationDataFrame`` to ``H2OIsolationForest`` as a parameter in order to
+``H2OGridSearch`` be able to evaluate produced models. The validation data frame has to have an extra column identifying
+whether the row represents an anomaly or not. The column can contain only two string values, where a value for the negative
+case, must be alphabetically smaller then the value for the positive case. E.g.: ``"0"``/``"1"``, ``"no"``/``"yes"``,
+``"false"``/``"true"``, etc.
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        Let's load a training and validation dataset at first:
+
+        .. code:: scala
+
+            import org.apache.spark.SparkFiles
+            spark.sparkContext.addFile("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate.csv")
+            spark.sparkContext.addFile("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate_anomaly_validation.csv")
+            val trainingDF = spark.read.option("header", "true").option("inferSchema", "true").csv(SparkFiles.get("prostate.csv"))
+            val validationDF = spark.read.option("header", "true").option("inferSchema", "true").csv(SparkFiles.get("prostate_anomaly_validation.csv"))
+
+        Create an algorithm instance, pass validation data frame, and specify a column identifying an anomaly:
+
+        .. code:: scala
+
+            import ai.h2o.sparkling.ml.algos.H2OIsolationForest
+            val algorithm = new H2OIsolationForest()
+            algorithm.setValidationDataFrame(validationDF)
+            algorithm.setValidationLabelCol("isAnomaly")
+
+        Define a hyper-space:
```

Comment:
Would be better to call this "hyper-parameter space"

Reply:
ok, will change, thanks!